saltgui/static/scripts/routes/route.js

```diff
@@ -39,4 +39,62 @@ class Route {
     return div;
   }
 
+  _runCommand(evt, targetString, commandString) {
+    this._runFullCommand(evt, "", targetString, commandString);
+  }
+
+  _runFullCommand(evt, targettype, targetString, commandString) {
+    this.router.commandbox._showManualRun(evt);
+    const target = document.querySelector("#target");
+    const command = document.querySelector("#command");
+    const targetbox = document.querySelector("#targetbox");
+
+    if(!targetString) targetString = "";
+    if(!commandString) commandString = "";
+
+    if(targettype) {
+      let tt = targettype;
+      // show the extended selection controls when
+      targetbox.style.display = "inherit";
+      if(tt !== "glob" && tt !== "list" && tt !== "compound" && tt !== "nodegroup") {
+        // we don't support that, revert to standard (not default)
+        tt = "glob";
+      }
+      TargetType.setTargetType(tt);
+    }
+
+    target.value = targetString;
+    command.value = commandString;
+    // the menu may become (in)visible due to content of command field
+    this.router.commandbox.cmdmenu.verifyAll();
+  }
+
+  _decodeArgumentsText(rawArguments) {
+    console.log(rawArguments);
+    const rawObject = rawArguments;
+    let argumentsText = "";
+
+    switch (typeof rawObject) {
+    case "undefined":
+      argumentsText = "";
+      break;
+    case "object": {      
```

Comment:
does anything bad happen without the { ... } ?

Reply:
Yes. I don't remember exact message but eslint was failing without it.